java/client/src/main/java/glide/api/models/configuration/RequestRoutingConfiguration.java

```diff
@@ -20,23 +22,31 @@ public interface Route {
         boolean isSingleNodeRoute();
     }
 
+    @RequiredArgsConstructor
+    @Getter
     public enum SimpleRoute implements Route {
         /** Route request to all nodes. */
-        ALL_NODES,
+        ALL_NODES(SimpleRoutes.AllNodes),
         /** Route request to all primary nodes. */
-        ALL_PRIMARIES,
+        ALL_PRIMARIES(SimpleRoutes.AllPrimaries),
         /** Route request to a random node. */
-        RANDOM;
+        RANDOM(SimpleRoutes.Random);
+
+        private final SimpleRoutes protobufMapping;
```

Comment:
Please, revert. I thought we already reverted this.

Reply:
Yes, you're using ordinal in `CommandManager` so it can be safely reverted.

Reply:
I thought we wanted this...

Reply:
No, it is not even used. We can't expose protobuf mapping to user
https://github.com/aws/glide-for-redis/pull/864#discussion_r1472743713